### PR TITLE
Fixes for CI pipeline pre-cxx11 pipeline

### PR DESCRIPTION
**Pull request by andi4191**
# Description
Added fixes required for CI pipeline

Fixes # (issue)

## Type of change

Please delete options that are not relevant and/or add your own.

- New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project (You can use the linters)
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests to verify my fix or my feature
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: py/setup.py [hunk 1]
```diff
@@ -239,7 +239,7 @@ def run(self):
         libraries=["torchtrt"],
         include_dirs=[
             dir_path + "torch_tensorrt/csrc", dir_path + "torch_tensorrt/include",
-            dir_path + "/../bazel-TRTorch/external/tensorrt/include",
```
Comment (narendasan) [inline py/setup.py:242]: We want to keep this, maybe try reordering

File: py/setup.py [hunk 2]
```diff
@@ -239,7 +239,7 @@ def run(self):
...
+            dir_path + "/../bazel-Torch-TensorRT/external/tensorrt/include",
```
Comment (narendasan) [inline py/setup.py:242]: This line is already included on the next line

File: py/setup.py (hunk 1 above)
Comment (andi4191) [reply py/setup.py]: fixed it. Let me find a workaround for this.

File: py/setup.py (hunk 2 above)
Comment (andi4191) [reply py/setup.py]: rectified it

**Review by narendasan — APPROVED** [review]
LGTM

**Pull request merged**